packages/tracing-internal/src/browser/browserTracingIntegration.ts

```diff
@@ -373,6 +380,16 @@ export function startBrowserTracingPageLoadSpan(
  * This will only do something if a browser tracing integration has been setup.
  */
 export function startBrowserTracingNavigationSpan(client: Client, spanOptions: StartSpanOptions): Span | undefined {
+  // TODO: should these have the same values? Does it matter? Should we just set one of the two?
+  getCurrentScope().setPropagationContext({
```

Comment:
we can probably also only set it on the current scope 🤔 but not sure...! doesn't hurt to do it this way for now.